test/integration/node/package.json

```diff
@@ -8,6 +8,6 @@
     "test-cjs": "node test.cjs"
   },
   "dependencies": {
-    "chart.js": "file:../package.tgz"
+    "chart.js": "workspace:*"
```

Comment:
This makes the integration tests use the sources instead of the shipping package?

Reply:
It makes it use the local package, but it will still use the built version